### PR TITLE
chage the message when no expense group imported

### PR DESCRIPTION
**Pull request by NileshPant1999**

### CODE REVIEW

File: src/app/netsuite/sync-export/sync/sync.component.ts [hunk 1]
```diff
@@ -109,31 +115,32 @@ export class SyncComponent implements OnInit {
...
+}
 
   ngOnInit() {
     const that = this;
     that.isLoading = true;
     that.workspaceId = +that.route.parent.snapshot.params.workspace_id;
 
     that.isExpensesSyncing = false;
-    this.updateLastSyncStatus();
+    this.updateLastSyncStatus().subscribe((res) => {
```
Comment (ashwin1111) [inline src/app/netsuite/sync-export/sync/sync.component.ts:139]: just `that.updateLastSyncStatus();` should work right?
you are assigning values to workspace and expenseGroupSettings at 2 places.

Comment (NileshPant1999) [reply src/app/netsuite/sync-export/sync/sync.component.ts:139]: yes didn't needed that removing it.

File: src/app/netsuite/sync-export/sync/sync.component.ts [hunk 2]
```diff
@@ -42,14 +42,20 @@ export class SyncComponent implements OnInit {
...
       if (res.results.filter(task => task.status === 'COMPLETE'  && task.type === 'FETCHING_EXPENSES').length === 1) {
-        that.updateLastSyncStatus();
-        that.isExpensesSyncing = false;
-        that.snackBar.open('Import Complete');
+        that.updateLastSyncStatus().subscribe((response) => {
+          that.isExpensesSyncing = false;
+          if (response[0].last_synced_at !== lastSyncedAt) {
+            that.snackBar.open('Import Complete');
+          } else {
+            that.snackBar.open('No New Expense Groups Imported');
```
Comment (ashwin1111) [inline src/app/netsuite/sync-export/sync/sync.component.ts:56]: Content tbd

**Review by ashwin1111 — APPROVED** [review]

**Review by Sravanksk — APPROVED** [review]

File: src/app/netsuite/sync-export/sync/sync.component.ts (hunk 2 above)
Comment (NileshPant1999) [reply src/app/netsuite/sync-export/sync/sync.component.ts:56]: @Shwetabhk

**Review by Shwetabhk — APPROVED** [review]

File: src/app/netsuite/sync-export/sync/sync.component.ts [hunk 3]
```diff
@@ -50,9 +51,18 @@ export class SyncComponent implements OnInit {
...
     ).subscribe((res) => {
       if (!res.results.length) {
-        that.updateLastSyncStatus();
+        that.updateLastSyncStatus().subscribe((response) => {
+          that.isExpensesSyncing = false;
+          if (response[0].last_synced_at !== lastSyncedAt) {
+            that.snackBar.open('Import Complete');
+          } else {
+            const expenseState = that.expenseGroupSettings.expense_state.toLowerCase().replace('_', ' ');
+            that.snackBar.open(`No new expense group imported kindly check your fyle account to see if there are any expenses in the ${expenseState} state`, null, {
```
Comment (Shwetabhk) [inline src/app/netsuite/sync-export/sync/sync.component.ts:60]: No new expense groups were imported. Kindly check your Fyle account to see if there are any expenses in the paid state.
```suggestion
            that.snackBar.open(`No new expense groups were imported. Kindly check your Fyle account to see if there are any expenses in the ${expenseState} state`, null, {
```

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**